modules/profile/manifests/tomcat.pp

```diff
@@ -17,6 +31,14 @@
   firewall::openport {'tomcat':
     dports          => $dports,
   }
+  
+  rsyslog::client { 'app' :
+    log_proto    => $log_proto,
+    log_port     => $log_port,
+    log_serv     => $log_serv,
+    apps         => $apps,
+  }
```

Comment:
oh... it should be
```
 rsyslog::client { 'tomcat' :
      log_serv     => $log_serv,
      log_name => '/var/log/tomcat/*.log',
      app_name => 'tomcat',
      severity => 'info'
 }

```
and 
```
rsyslog::client { 'httpd' :
      log_serv     => $log_serv,
      log_name => '/var/log/httpd/*.log',
      app_name => 'httpd',
      severity => 'info'
 }
```

Reply:
I want to have something like that ^^^

Reply:
and in rsyslog::client you need to create file for each app in  /etc/rsyslog.d/
e.g:
/etc/rsyslog.d/httpd.conf
/etc/rsyslog.d/tomcat.conf